css/dashboards/gui.css

```diff
@@ -816,3 +820,63 @@
 .highcharts-dashboards-row .highcharts-dashboards-cell:focus {
     outline: 5px auto -webkit-focus-ring-color;
 }
+
+/* Dropdown */
+ul.highcharts-dashboards-dropdown-content {
+    display: flex;
+    flex-direction: column;
+    gap: 4px;
+    background: var(--highcharts-dashboards-main-background-color);
+    box-sizing: border-box;
+    padding: 16px 8px;
+    margin-top: 4px;
+    border: 1px solid var(--highcharts-dashboards-dropdown-border-color);
+    box-shadow: 0 10px 24px var(--highcharts-dashboards-dropdown-box-shadow-color);
+    list-style-type: none;
+}
+
+ul > .highcharts-dashboards-dropdown {
+    position: relative;
+    font-family: Arial, Helvetica, sans-serif;
+    color: var(--highcharts-dashboards-main-font-color);
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    width: 100%;
+}
+
+.highcharts-dashboards-dropdown > button.highcharts-dashboards-dropdown-button {
+    width: 100%;
+    color: var(--highcharts-dashboards-main-font-color);
+    box-sizing: border-box;
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+    align-items: center;
+    padding: 6px 8px;
+    margin: 2px;
+    gap: 2px;
+    background: var(--highcharts-dashboards-main-background-color);
+    border: 1px solid var(--highcharts-dashboards-dropdown-border-color);
+    border-radius: 2px;
+    flex: none;
+    order: 0;
+    align-self: stretch;
+    flex-grow: 0;
+}
+
+button.highcharts-dashboards-select-option-button {
+    display: flex;
+    flex-direction: row;
```

Comment:
Do we need it for the button?

Reply:
in the button, there are several elements, which needs to be positioned as well, so yes, we need it :)